steps/mentor-calibre-drc/configure.yml

```diff
@@ -24,7 +24,11 @@ outputs:
 #-------------------------------------------------------------------------
 
 commands:
+  - if [ $full_chip == True ]; then export design_name={design_name}_filled; fi
   - envsubst < drc.runset.template > drc.runset
+  - cp inputs/adk/{drc_rule_deck} {drc_rule_deck}
+  - if [ $full_chip == True ]; then sed -i 's/\/\/#DEFINE FULL_CHIP/#DEFINE FULL_CHIP/' {drc_rule_deck}; fi
+  - if [ $full_chip == True ]; then sed -i 's/\/\/#DEFINE WITH_SEALRING/#DEFINE WITH_SEALRING/' {drc_rule_deck}; fi
```

Comment:
Instead of doing this, we should make a version of the DRC rule deck called "calibre-drc-chip.rule" that has all options set for full chip. Then you just set the "drc_rule_deck" mflowgen parameter to that when you want to run full chip drc.